spantest/emulator.go

```diff
@@ -46,7 +46,7 @@ type EmulatorFixture struct {
 }
 
 // NewEmulatorFixture creates a test fixture for a containerized Spanner emulator.
-func NewEmulatorFixture(t testing.TB) Fixture {
+func NewEmulatorFixture(t testing.TB) FixtureWithProtoDescriptorSupport {
```

Comment:
Isn't this a breaking change?

Reply:
Should be non-breaking to extend (or return a larger) interface. 👍

Reply:
No it should not be breaking since it's extending the existing interface as @radhus mentions.